server/serverjs/cubefn.js

```diff
@@ -272,6 +275,7 @@ export const CSVtoCards = (csvString, carddb) => {
         const nonPromo = potentialIds.find(carddb.reasonableId);
         const first = potentialIds[0];
         card.cardID = matchingSetAndNumber || matchingSet || nonPromo || first;
+        card.name = overrideName ?? name;
```

Comment:
We want the name property to be null if there isn't an override that way it picks up the correct name from details.